fe/src/main/jflex/sql_scanner.flex

```diff
@@ -239,6 +239,7 @@ import org.apache.doris.qe.SqlModeHelper;
         keywordMap.put("load", new Integer(SqlParserSymbols.KW_LOAD));
         keywordMap.put("local", new Integer(SqlParserSymbols.KW_LOCAL));
         keywordMap.put("location", new Integer(SqlParserSymbols.KW_LOCATION));
+        keywordMap.put("materialized", new Integer(SqlParserSymbols.KW_MATERIALIZED));
```

Comment:
Can you explain why add this for this PR, I noticed you delete this keyword in `sql_parser`

Reply:
I didn't delete this keyword “materialized”. The keywords “materialized” are not sorted in alphabetical order, so I only put this keyword in the position it should be